grails-app/views/productAssociation/create.gsp

```diff
@@ -95,7 +95,7 @@
 									<label for="dateCreated"><warehouse:message code="productAssociation.dateCreated.label" default="Date Created" /></label>
 								</td>
 								<td valign="top" class="value ${hasErrors(bean: productAssociationInstance, field: 'dateCreated', 'errors')}">
-									<g:datePicker name="dateCreated" precision="minute" value="${productAssociationInstance?.dateCreated}"  />
+									<g:datePicker name="dateCreated" value="${productAssociationInstance?.dateCreated}"  />
```

Comment:
This change wasn't required, but now that we're using java.time classes it can figure out the precision on its own based on the type (LocalDate is day precision, everything else is minute) and so this field no longer needs to be provided.